doc/tutorials/using_windows_as_uos.rst

```diff
@@ -72,43 +72,82 @@ Create a raw disk
 Run these commands on the Service VM::
 
    $ sudo apt-get install qemu-utils
-   $ mkdir /home/acrn/img
-   $ cd /home/acrn/img
+   $ mkdir /home/acrn/work
+   $ cd /home/acrn/work
    $ qemu-img create -f raw win10-ltsc.img 30G
 
 Prepare the script to create an image
 -------------------------------------
 
-#. Copy ``/usr/share/acrn/samples/nuc/launch_win.sh`` to ``install_win.sh``::
-
-   $ cp /usr/share/acrn/samples/nuc/launch_win.sh /home/acrn/img/install_win.sh
-
-
-#. Edit the ``acrn-dm`` command line in ``install_win.sh`` as follows:
-
-   .. note:: Make sure you use GVT-g ``-s 2,pci-gvt -G "$2"`` in the
-      ``acrn-dm`` command line. Currently, we do not support creating a
-      windows image by GVT-d.
-
-   - Change ``-s 3,virtio-blk,./win10-ltsc.img`` to your path to the Windows
-     10 image.
+#. Refer to :ref:`gpu-passthrough` to enable GVT-d GOP feature. Then
+   copy above ``.iso`` files and the build ``OVMF.fd`` to
+   ``/home/acrn/work``
+#. Prepare WaaG install script
+
+.. code-block:: bash
+
+   vim /home/acrn/work/install_win.sh
+
+   #!/bin/bash
+   function launch_win()
+   {
+   vm_name=win_vm$1
+   #check if the vm is running or not
+   vm_ps=$(pgrep -a -f acrn-dm)
+   result=$(echo $vm_ps | grep "${vm_name}")
+   if [[ "$result" != "" ]]; then
+     echo "$vm_name is running, can't create twice!"
+     exit
+   fi
+   echo "8086 9ded" > /sys/bus/pci/drivers/pci-stub/new_id
+   echo "0000:00:14.0" > /sys/bus/pci/devices/0000:00:14.0/driver/unbind
+   echo "0000:00:14.0" > /sys/bus/pci/drivers/pci-stub/bind
+   echo "8086 3ea0" > /sys/bus/pci/drivers/pci-stub/new_id
+   echo "0000:00:02.0" > /sys/bus/pci/devices/0000:00:02.0/driver/unbind
+   echo "0000:00:02.0" > /sys/bus/pci/drivers/pci-stub/bind
+   #for memsize setting
+   mem_size=4096M
+   acrn-dm -A -m $mem_size -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
+     -s 2,passthru,0/2/0,gpu \
+     -s 8,virtio-net,tap0 \
+     -s 4,virtio-blk,/home/acrn/work/win10-ltsc.img
+     -s 5,ahci,cd:./home/acrn/work/windows.iso \
```

Comment:
Do we really need the dot at the beginning of the path? Or should this be
```suggestion
     -s 5,ahci,cd:/home/acrn/work/windows.iso \
```
?

Reply:
Forget to remove dot when update it to absolute path.
Thanks!